src/EditorFeatures/Core.Cocoa/NavigationCommandHandlers/FindBaseSymbolsCommandHandler.cs

```diff
@@ -49,8 +48,7 @@ protected override bool TryExecuteCommand(int caretPosition, Document document,
             var streamingPresenter = base.GetStreamingPresenter();
             if (streamingPresenter != null)
             {
-                // Fire and forget.  So no need for cancellation.
-                _ = StreamingFindBaseSymbolsAsync(document, caretPosition, streamingPresenter, CancellationToken.None);
+                _ = StreamingFindBaseSymbolsAsync(document, caretPosition, streamingPresenter);
```

Comment:
all the cocoa code operates ina  fire-and-forget manner, where the ui thread is relinguished back to the user, and the results are streamed into the FAR window.  AS such, we do *not* pass a cancellation token along here.  And in teh fire-and-forget method, we grab the CT from teh presenter and use that to control teh search.